R/util-MakeBounds.R

```diff
@@ -31,9 +31,19 @@ MakeBounds <- function(
   dSnapshotDate = unique(dfResults$SnapshotDate),
   strStudyID = unique(dfResults$StudyID)
 ) {
-  stop_if_empty(strMetrics)
-  stop_if_empty(dSnapshotDate)
-  stop_if_empty(strStudyID)
+  # stop_if(cnd = !length(strMetrics), message = "`strMetrics` must not be `NULL`.")
+  # stop_if(cnd = !length(dSnapshotDate), message = "`dSnapshotDate` must not be `NULL`.")
+  # stop_if(cnd = !length(strStudyID), message = "`strStudyID` must not be `NULL`.")
+
+  stop_if_any(
+    conditions = exprs(!length(strMetrics), !length(dSnapshotDate), !length(strStudyID)),
```

Comment:
looks like we need to add `rlang exprs` to the dependencies. was getting some errors when running workflows and tests.

Reply:
I want to say this will only be necessary for if we go with this `stop_if_any()` that accepts all the conditions & messages at once, the single call one I dont think its necessary as I use `enquo()` which was already in the `utils-tidy-eval.R`